design-documents/graph-ql/coverage/gift-registry.graphqls

```diff
@@ -0,0 +1,278 @@
+type Customer {
+    gitft_registry_list: [GiftRegistry]
```

Comment:
Do we typically use hungarian notation for type names? `list` seems fairly clear from the type def

Reply:
This is to differentiate from a query returning single gift registry.
Does `gift_registries` sound better to you?

Reply:
> Does gift_registries sound better to you?

Yeah definitely - I like that!